package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -1083,47 +1228,52 @@ def save_table(self, filename="hbond_table.pickle"):
             self.generate_table()
         cPickle.dump(self.table, open(filename, 'wb'), protocol=cPickle.HIGHEST_PROTOCOL)
 
+    def _has_timeseries(self):
+        has_timeseries = (self._timeseries is not None)
```

Comment:
I like them for readability but I am happy to removed them – if you didn't care you wouldn't have taken the time to comment.